invenio_cli/commands/packages.py

```diff
@@ -37,6 +37,44 @@ def install_packages(packages):
 
         return steps
 
+    @staticmethod
+    def outdated_packages():
+        """Steps to show outdated packages.
+
+        It is a class method since it does not require any configuration.
+        """
+        prefix = ['pipenv', 'update']
+        cmd = prefix + ['--outdated']
+
+        steps = [
+            CommandStep(
+                cmd=cmd,
+                env={'PIPENV_VERBOSITY': "-1"},
+                message="Checking outdated packages..."
+            )
+        ]
+
+        return steps
+
+    @staticmethod
+    def update_packages(package):
```

Comment:
moderate: you can make the parameter optional:
```suggestion
    def update_packages(package=None):
```

That way above, when calling you do not need to pass `""`, in line *invenio_cli/cli/packages.py:88* would be:
```
steps = PackagesCommands.update_packages()
```